docs/how-to/h-deploy-azure.md

```diff
@@ -0,0 +1,242 @@
+For more guidance on production environment sizing, see the [Requirements page](/t/charmed-kafka-reference-requirements/10563).
+You can find more information about the available instance types in the [Azure documentation](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/overview).
+
+Deploy and integrate Kafka and ZooKeeper, for example:
+
+```shell
+juju deploy zookeeper -n3 --channel 3/stable
+juju deploy kafka -n3 --constraints "instance-type=Standard_A4_v2" --channel 3/stable
+juju integrate kafka zookeeper
+```
+
+We also recommend to deploy a [Data Integrator](https://charmhub.io/data-integrator) for creating an admin user to manage the content of the Kafka cluster:
+
+```shell
+juju deploy data-integrator admin --channel edge \
+  --config extra-user-roles=admin \
+  --config topic-name=admin-topic
+```
+
+And integrate it with the Kafka application:
+
+```shell
+juju integrate kafka admin
+```
+
+For more information on Data Integrator and how to use it, please refer to the [how-to manage applications](/t/charmed-kafka-how-to-manage-app/10285) guide.
+
+## Clean up
+
+[note type="caution"]
+Always clean Azure resources that are no longer necessary! Abandoned resources are tricky to detect and they can become expensive over time.
+[/note]
+
+To destroy the Juju controller and remove the Azure instance (Warning: all your data will be permanently removed):
+
+```shell
+juju destroy-controller <CONTROLLER_NAME> --destroy-all-models --destroy-storage --force
+```
+
+> Use `juju list-controllers` to retrieve the names of the controllers that have been registered to your local client. 
+
+Next, check and manually delete all unnecessary Azure VM instances, to show the list of all your Azure VMs run the following command (make sure to use the correct region): 
+```shell
+az resource list
+```
+
+List your Juju credentials with the `juju credentials` command:
+
+```shell
+Client Credentials:
+Cloud        Credentials
+azure        NAME_OF_YOUR_CREDENTIAL
+```
+
+Remove Azure CLI credentials from Juju:
+
+```shell
+juju remove-credential azure NAME_OF_YOUR_CREDENTIAL
+```
+
+After deleting the credentials, the `interactive` process may still leave the role resource and its assignment hanging around. 
+We recommend checking if these are still present by running:
+
+```shell
+az role definition list --name <AZURE_ROLE>
+```
+
+To get the full list, use it without specifying the `--name` argument. 
+
+You can check whether you still have a 
+role assignment bound to `<AZURE_ROLE>` registered by using:
+
+```shell
+az role assignment list --role <AZURE_ROLE>
+```
+
+If there is an unwanted role left, you can remove the role assignment first and then the role itself with the following commands:
+
+```shell
+az role assignment delete --role <AZURE_ROLE>
+az role definition delete --name <AZURE_ROLE>
+```
+
+Finally, logout Azure CLI user credentials to prevent any credential leakage:
+
+```shell
+az logout 
+```
+
```

Comment:
Misses an integration with COS

Reply:
we have a dedicated how-to, for integrating with COS: https://charmhub.io/kafka/docs/h-enable-monitoring